src/material/datepicker/calendar-body.ts

```diff
@@ -77,43 +83,81 @@ export class MatCalendarBody implements OnChanges, OnDestroy {
   @Input() rows: MatCalendarCell[][];
 
   /** The value in the table that corresponds to today. */
-  @Input() todayValue: number;
+  @Input()
+  get todayValue(): number { return this._todayValue; }
```

Comment:
This component doesn't need coercion, because it's internal to the datepicker calendar.